docs/mobile/exercises/README.md

```diff
@@ -231,11 +224,19 @@ tabs:
 
 Tabs appear in the top right of the exercise and serve as a way to display
 datasets or files. The appropriate asset must be defined in the [`assets`
-section of the course manifest](../repo-structure.md#assets-metadata). A tab
+section of the course manifest](/mobile/repo-structure.md#assets-metadata). A tab
 field has two subfields: `key`, which is the alias for the asset as defined in
 the manifest, and `type`, which specifies how to render the asset. Currently
 only `file` and `table` are supported.
 
+An example of a `dataset` rendered as a `table`:
+
+![A tab from the exercise screen](/images/mobile/tabs-dataset-table-small.png) ![Opened dataset tab](/images/mobile/tabs-dataset-table-open-small.png)
+
+An example of a rendered `file` tab:
+
+![Opened file tab](/images/mobile/tabs-file-open-small.png)
+
 # Summary of available blocks
```

Comment:
For the table:
- In a Multiple Choice question, the question is REQUIRED

Reply:
right